R/api.R

```diff
@@ -54,8 +54,15 @@ model_compile <- function(data, pretty = FALSE) {
 
 ##' @porcelain GET /support/runner-ode => json
 support_runner_ode <- function() {
-  code <- odin::odin_js_bundle(NULL, include_support = TRUE)$support
+  code <- read_string(system_file("js/odin.js", "odin"))
   ## We add "odinjs;" after the code here so that a JS `eval()` around
   ## it returns the object.
   scalar(paste0(code, "odinjs;"))
 }
+
+
+##' @porcelain GET /support/runner-discrete => json
+support_runner_discrete <- function() {
```

Comment:
Yes, this is all the stuff from dust-js - it ended up being pretty nasty to get the two combined nicely, but if this ends up being hard to handle in the front end we can work something out.

It can be requested once on app load, yes. This is the reason we put the endpoint at support/runner-ode (rather than support/runner) as I think at that point I knew they'd be likely different

Reply:
Makes sense, I'll make a ticket to fetch that second runner in WODIN